internal/btf/btf.go

```diff
@@ -599,8 +589,29 @@ func (h *Handle) FD() int {
 
 // Map is the BTF for a map.
 type Map struct {
+	name       string
```

Comment:
I don't understand why you add these fields to `Map`?

Reply:
Nice catch, this was a leftover! :+1: